test/Makefile

```diff
@@ -176,6 +191,48 @@ run: ginkgo
 		-default-cls-service-account-name=$(DEFAULT_CLS_SERVICE_ACCOUNT_NAME) \
 		-standby-cls-service-account-name=$(STANDBY_CLS_SERVICE_ACCOUNT_NAME)
 
+.PHONY: run-perf
+run-perf: ginkgo
+		@[ "${CREDS_FILE}" ] && echo "Using credentials from ${CREDS_FILE}" || \
+			( echo "A credentials file is required to run E2E tests, please re-run the make target with CREDS_FILE=<PathToCredentialsFile>"; exit 1 )
+		@[ "${BSL_BUCKET}" ] && echo "Using bucket ${BSL_BUCKET} to store backups from E2E tests" || \
+			(echo "Bucket to store the backups from E2E tests is required, please re-run with BSL_BUCKET=<BucketName>"; exit 1 )
+		@[ "${CLOUD_PROVIDER}" ] && echo "Using cloud provider ${CLOUD_PROVIDER}" || \
+			(echo "Cloud provider for target cloud/plugin provider is required, please rerun with CLOUD_PROVIDER=<aws,azure,kind,vsphere>"; exit 1)
+	@$(GINKGO) -v $(FOCUS_STR) $(SKIP_STR) ./perf -- -velerocli=$(VELERO_CLI) \
+		-velero-image=$(VELERO_IMAGE) \
```

Comment:
Consider to move out shared parameters with run-e2e, then make it reusable to run-perf.

Reply:
Adjusted